assets/scripts/airport.js

```diff
@@ -1346,6 +1346,7 @@ function airport_init() {
   airport_load('sbgr', "beginner", "Aeroporto Internacional de São Paulo/Guarulhos");
   airport_load('tncm', "easy", "Princess Juliana International Airport");
   airport_load('uudd', "easy", "Moscow Domodedovo Airport");
+  airport_load('vobl', "hard", "Kempegowda International Airport Bengaluru");
```

Comment:
in `v3`, this line will need to go in the `airportLoadList.js` file. see the comments in https://github.com/zlsa/atc/blob/gh-pages/src/assets/scripts/airport/airportLoadList.js for more information here.

We have #719 and n8rzz#88 which will eliminate the need for any npm commands when adding new airports/airlines/aircraft.